src/android/bugsnag-plugin-android-cocos2dx/build.gradle

```diff
@@ -15,12 +15,6 @@ android {
     defaultConfig {
         minSdkVersion 16 // Minimum version supported by Cocos2dx
         versionName '1.0.0'
-        ndkVersion = "16.1.4479499"
-
-        // build config doesn't automatically include these values anymore but they're used
-        // by the cocos2dx plugin - add them manually
-        buildConfigField 'int', 'VERSION_CODE', "1"
-        buildConfigField 'String', 'VERSION_NAME', "\"1.0.0\""
```

Comment:
Could we create constants in the plugin class instead? That would avoid the need for anyone on AGP >4.1 to manually add these lines in.

https://github.com/bugsnag/bugsnag-cocos2dx/blob/next/src/android/bugsnag-plugin-android-cocos2dx/src/main/java/com/bugsnag/android/BugsnagCocos2dxPlugin.java#L64

Reply:
That seems reasonable! :writing_hand: